core/tests/protractor/editorAndPlayer.js

```diff
@@ -205,6 +205,7 @@ describe('Full exploration editor', function() {
       editor.setTitle('Testing multiple rules');
       editor.setCategory('Answer Groups');
       editor.setObjective('To assess happiness.');
+      editor.previewSummaryTile();
```

Comment:
Ah, ok, interesting. In that case I would suggest renaming to openAndCloseSummaryTilePreview and keeping it as one method, since this gives a better indication of the actions involved.